x-pack/plugins/synthetics/server/synthetics_service/synthetics_service.ts

```diff
@@ -273,6 +287,30 @@ export class SyntheticsService {
     }
   }
 
+  async editConfig(monitorConfig: SyntheticsConfig) {
+    const monitors = this.formatConfigs([monitorConfig]);
+
+    this.apiKey = await this.getApiKey();
+
+    if (!this.apiKey) {
+      return null;
```

Comment:
Wondering if we should log a warning for this. This is a common occurrance when monitor management is disabled, so it doesn't necessary represent an error, but wondering if users might get confused and Kibana logging might help.

Reply:
this isn't added in this PR. we can definitely improve it though.